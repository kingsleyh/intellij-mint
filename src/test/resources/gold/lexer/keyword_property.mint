component A {

  property name : String = ""

  fun render : Html {
    </div>
  }

}