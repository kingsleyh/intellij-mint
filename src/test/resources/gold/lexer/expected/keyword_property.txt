component ('component')
WHITE_SPACE (' ')
IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
property ('property')
WHITE_SPACE (' ')
IDENT ('name')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('String')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
STRING_LITERAL ('""')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
IDENT ('render')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Html')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
</ ('</')
IDENT ('div')
> ('>')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')